mne/epochs.py

```diff
@@ -2034,14 +2034,17 @@ def filename(self):
 
     def __repr__(self):
         """Build string representation."""
-        s = f" {len(self.events)} events "
+        s = f"{len(self.events)} events "
         s += "(all good)" if self._bad_dropped else "(good & bad)"
-        s += f", {self.tmin:g} – {self.tmax:g} s"
-        s += ", baseline "
+        s += f", {self.tmin:.3f}".rstrip("0").rstrip(".")
+        s += f" – {self.tmax:.3f}".rstrip("0").rstrip(".")
```

Comment:
this `rstrip` stuff isn't necessary if you use `g` format specifier:

```pycon
In [1]: f"{1.23456:.3g}"
Out[1]: '1.23'
In [2]: f"{1.00006:.3g}"
Out[2]: '1'
```

```suggestion
        s += f", {self.tmin:.3g}"
        s += f" – {self.tmax:.3g}"
```

Reply:
Nice! I think one could now also combine these two lines into one?

Reply:
That's not correct, `:.3g` shows three significant digits, whereas `:.3f` shows three decimal places. We want the latter here.

Reply:
ah!

Reply:
oops, you're correct; I always forget that "g" means significant digits because in the docs they call it "general format"